pluto/src/tree/core.ts

```diff
@@ -69,17 +71,25 @@ export const flatten = ({
   expanded,
   depth = 0,
   sort = true,
+  path = "",
 }: FlattenProps): FlattenedNode[] => {
   // Sort the first level of the tree independently of the rest
   if (depth === 0 && sort) nodes = nodes.sort((a, b) => a.name.localeCompare(b.name));
   const flattened: FlattenedNode[] = [];
   nodes.forEach((node, index) => {
+    const nextPath = `${path + node.key}/`;
```

Comment:
If you're using string formatting, might as well not use concatenation i.e. `${path}${node.key}/`